docs/source/quantization-support.rst

```diff
@@ -296,16 +296,16 @@ then be quantized.
     BNReLU2d
     BNReLU3d
 
-torch.nn.intrinsic.qat
-~~~~~~~~~~~~~~~~~~~~~~
-.. automodule:: torch.nn.intrinsic.qat
-.. automodule:: torch.nn.intrinsic.qat.modules
+torch.ao.nn.intrinsic.qat
```

Comment:
I feel ideally maybe we should change the docs in one PR to avoid inconsistent docs, but looks like we already have a partially up to date doc

Reply:
The docs will have to be readdressed a lot -- there are a lot of inconsistencies in the documentation. This change is here because it is needed to pass the tests though